inc/admin/class-envato-market-admin.php

```diff
@@ -1309,6 +1312,259 @@ public function ajax_remove_item() {
 		}
 
 		/**
+		 * AJAX handler for performing a healthcheck of the current website.
+		 *
+		 * @since 2.0.6
+		 * @codeCoverageIgnore
+		 */
+		public function ajax_healthcheck() {
+			if ( ! check_ajax_referer( self::AJAX_ACTION, 'nonce', false ) ) {
+				status_header( 400 );
+				wp_send_json_error( 'bad_nonce' );
+			} elseif ( 'POST' !== $_SERVER['REQUEST_METHOD'] ) {
+				status_header( 405 );
+				wp_send_json_error( 'bad_method' );
+			}
+
+			$limits = $this->get_server_limits();
+
+			wp_send_json_success( array(
+				'limits' => $limits
+			) );
+		}
+
+	  /**
+	   * AJAX handler for performing a healthcheck of the current website.
+	   *
+	   * @since 2.0.6
+	   * @codeCoverageIgnore
+	   */
+	  public function get_server_limits() {
+		  $limits = [];
+
+		  // Check memory limit is > 256 M
+		  try {
+			  $memory_limit         = wp_convert_hr_to_bytes( ini_get( 'memory_limit' ) );
+			  $memory_limit_desired = 256;
+			  $memory_limit_ok      = $memory_limit < 0 || $memory_limit >= $memory_limit_desired * 1024 * 1024;
+			  $memory_limit_in_mb   = $memory_limit < 0 ? 'Unlimited' : floor( $memory_limit / ( 1024 * 1024 ) ) . 'M';
```

Comment:
This might be too nitpicky, but we display the memory limit as e.g. "128M" here, and the error message says "please set your PHP memory limit to at least 256MB". Should we try to standardise the use of "M" vs. "MB" to make it clear that we're talking about the same thing?

Reply:
Seems `M` is the standard here, so going with that: https://github.com/envato/wp-envato-market/pull/214/commits/fe4c70d9826cd00b4f65cad0fcd764365f41b370